infra/terraform/modules/data_access/gh_webhook_handler/index.py

```diff
@@ -0,0 +1,91 @@
+import hmac
```

Comment:
Yes, `github_webhooks` is a better name for the module. I'm not sure about `index.py` - does the file structure need to look like this? Could you have the code in `infra/terraform/modules/data_access/github_webhooks.py`? Or if the current file structure is needed, then maybe the `index.py` file should be called `publish_to_sns.py` and the `handler` function can keep its name?